issue #8515 Spelling mistakes found checking `en-GB` to `en-US`

albert-github opened the pull request:
Spelling corrections, `en-GB` -> `en-US`

albert-github committed:
issue #8515 Spelling mistakes found checking `en-GB` to `en-US`

Spelling corrections, `en-GB` -> `en-US`

afabri commented on Nef_2/include/CGAL/Nef_2/PM_point_locator.h:
```diff
@@ -536,7 +536,7 @@ class PM_point_locator : public
   heuristics. Therefore worst case bounds are not very expressive. The
   query operations take up to linear time for subsequent query
-  operations though they are better in practise. They trigger a one-time
+  operations though they are better in practize. They trigger a one-time
```
```suggestion
  operations though they are better in practice. They trigger a one-time
```

afabri commented on Number_types/include/CGAL/Sqrt_extension/Algebraic_extension_traits.h:
```diff
@@ -22,7 +22,7 @@ template <class COEFF, class ROOT, class ACDE_TAG,class FP_TAG>
 /* needed to 'add up' sqrt_extensions in iterator range such that all roots
    are collected in order to keep operation time minimal all scalar coeffs
-   are set to 1 by standardise.
+   are set to 1 by standardize.
```
looks strange.  Maybe "by default"?

afabri replied:
It is only in a comment so let's just ignore it

albert-github replied:
Indeed, changed it

albert-github committed:
Update Nef_2/include/CGAL/Nef_2/PM_point_locator.h

You found that problem quickly!!! Thanks. 
Must have been the sheer amount of changes.

Co-authored-by: Andreas Fabri <andreas.fabri@geometryfactory.com>

afabri reviewed (approved):
Thanks a lot, Albert

afabri commented:
@sloriot @lrineau I would just merge it.

albert-github committed:
issue #8515 Spelling mistakes found checking `en-GB` to `en-US`

After review.
